### PR TITLE
Np 10167/referral menu

### PR DESCRIPTION
**Pull request by AlexeyGreer**
Screenshot 2023-11-27 at 2 00 05 PM

### CODE REVIEW

File: src/foam/nanos/referral/ReferUserView.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      border-style: dashed;
+      border-color: $primary400;
+      border-width: 2px;
+      padding: 1.5rem;  
+      text-align: center;
+      border-width: thin;
+    }
+    ^header{
+      font-size:3rem;
+      color: #1A6B54;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/referral/ReferUserView.js]:
```suggestion
      color: $primary500;
```

File: src/foam/nanos/referral/pom.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ foam.POM({
 ],
   files: [
     { name: "ReferralCode",                       flags: "js|java" },
-    { name: "CreateReferralCode",                 flags: "js|java" }
+    { name: "CreateReferralCode",                 flags: "js|java" },
+    { name: "ReferUserView",                   flags: "web" }
```
Comment (jlhughes) [inline src/foam/nanos/referral/pom.js]: Kevin won't like this.

Comment (AlexeyGreer) [reply src/foam/nanos/referral/pom.js]: What is the issue?

Comment (jlhughes) [reply src/foam/nanos/referral/pom.js]: Spacing.  I don't use a proportional font, so not sure how many spaces you need.

**Review by jlhughes — APPROVED** [review]

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**